AB_environments/AB_baseline.requirements.txt

```diff
@@ -0,0 +1,44 @@
+# Copy-paste from ci/environment.yml
+coiled>=0.2.54
+numpy==1.26.2
+pandas==2.1.4
+dask-labextension==7.0.0
+dask-ml==2023.3.24
+fsspec==2023.12.1
+s3fs==2023.12.1
+gcsfs==2023.12.1
+pyarrow==15.0.0
+jupyterlab==4.0.9
+lz4==4.3.2
+ipywidgets==8.1.1
+numba==0.58.1
+scikit-learn==1.3.2
+ipycytoscape==1.3.3
+click==8.1.7
+xarray==2023.12.0
+zarr==2.16.1
+cftime==1.6.3
+msgpack==1.0.7
+cloudpickle==3.0.0
+tornado==6.3.3
+toolz==0.12.0
+zict==3.0.0
+xgboost==1.7.6
+optuna==3.5.0
+scipy==1.11.4
+sqlalchemy==2.0.23
+pynvml==11.5.0
+bokeh==3.3.2
+gilknocker==0.4.1
+# End copy-paste
+
+# Make sure you install dask and distributed either both from pip or both from
+# conda. You may alternatively point to your own git fork (but make sure you
+# sync'ed tags!)
```

Comment:
This comment looks obsolete?

Reply:
https://github.com/coiled/benchmarks/pull/1447/commits/1d9a784f673b3b56a042fcbe92ff04db1df08569